homework/index.js

```diff
@@ -30,18 +30,100 @@
     return elem;
   }
 
+  function createInfoDiv(repoDiv, selectRepo) {
+    const tableDiv = createAndAppend('div', repoDiv, { class: 'left-div' });
+    const infoTable = createAndAppend('table', tableDiv, { class: 'contributor-table' });
+    const tableBody = createAndAppend('tbody', infoTable);
+    const rowName = createAndAppend('tr', tableBody);
+    const repoName = createAndAppend('td', rowName, { text: 'Repository:', class: 'repo-name' });
+    createAndAppend('a', repoName, {
+      target: '_blank',
+      href: selectRepo.html_url,
+      text: selectRepo.name,
+    });
+    const rowDesc = createAndAppend('tr', tableBody);
+    createAndAppend('td', rowDesc, {
+      text: `Description: ${selectRepo.description}`,
+      class: 'repo-desc',
+    });
+    const rowFork = createAndAppend('tr', tableBody);
+    createAndAppend('td', rowFork, {
+      text: `Forks: ${selectRepo.forks}`,
+      class: 'repo-Fork',
```

Comment:
Always use kebab-case for CSS class names and ids (lower case letter and hyphens to separate words).